src/ShapeShiftController.ts

```diff
@@ -95,19 +95,21 @@ export class ShapeShiftController extends BaseController<ShapeShiftConfig, Shape
 		this.defaultConfig = { interval: 3000 };
 		this.defaultState = { shapeShiftTxList: [] };
 		this.initialize();
+		this.poll();
 	}
 
 	/**
-	 * Sets a new polling interval
+	 * starts a new polling interval
```

Comment:
Very minor comment, we have documentation starting with capital letter, this (and other changes in the same method in other controllers) changes this